src/gluonnlp/models/model_zoo_checksums/gpt2.txt

```diff
@@ -1,15 +1,20 @@
-gpt2_124M/model-fac1f39c.yml       fac1f39c804e324c69162b9b37bd24ab98241612    424
-gpt2_124M/model_lm-99b90604.params 99b9060488b4542ccd045c28401da10a3158ca80    497771820
-gpt2_124M/gpt2-396d4d8e.merges     396d4d8ec90cb02f4d56e049e0e4add868bcd943    456318
-gpt2_124M/gpt2-9dc62091.vocab      9dc620913410d5ec1a988abf852891e1c9f0f649    558055
-gpt2_124M/model-bfed311d.params    bfed311d5c980ba475f90ccf7f536d25c3b40386    497769466
-gpt2_355M/model-2aea05ff.yml       2aea05ff1e67ef816b3f824102da8b7b1292a620    425
-gpt2_355M/model_lm-eed0e964.params eed0e964f4222823a557acfee2c106f228ce0188    1419317644
-gpt2_355M/gpt2-396d4d8e.merges     396d4d8ec90cb02f4d56e049e0e4add868bcd943    456318
-gpt2_355M/gpt2-9dc62091.vocab      9dc620913410d5ec1a988abf852891e1c9f0f649    558055
-gpt2_355M/model-81dee612.params    81dee612413733899f6e5fbbeac91da781805e1b    1419312986
-gpt2_774M/model-c9555788.yml       c95557880783ec4f94b09b5b045c8d9e9a198e4d    425
-gpt2_774M/model_lm-cfbfa641.params cfbfa6419aaf1eae480fba5a1a7c8ea6096d43d6    3096157676
-gpt2_774M/gpt2-396d4d8e.merges     396d4d8ec90cb02f4d56e049e0e4add868bcd943    456318
-gpt2_774M/gpt2-9dc62091.vocab      9dc620913410d5ec1a988abf852891e1c9f0f649    558055
-gpt2_774M/model-9917e24e.params    9917e24e89c651793adea69042d6cceddfc7973c    3096150714
+gpt2_124M/model-fac1f39c.yml        fac1f39c804e324c69162b9b37bd24ab98241612    424
+gpt2_124M/model_lm-99b90604.params  99b9060488b4542ccd045c28401da10a3158ca80    497771820
+gpt2_124M/gpt2-396d4d8e.merges      396d4d8ec90cb02f4d56e049e0e4add868bcd943    456318
+gpt2_124M/gpt2-9dc62091.vocab       9dc620913410d5ec1a988abf852891e1c9f0f649    558055
+gpt2_124M/model-bfed311d.params     bfed311d5c980ba475f90ccf7f536d25c3b40386    497769466
+gpt2_355M/model-2aea05ff.yml        2aea05ff1e67ef816b3f824102da8b7b1292a620    425
+gpt2_355M/model_lm-eed0e964.params  eed0e964f4222823a557acfee2c106f228ce0188    1419317644
+gpt2_355M/gpt2-396d4d8e.merges      396d4d8ec90cb02f4d56e049e0e4add868bcd943    456318
+gpt2_355M/gpt2-9dc62091.vocab       9dc620913410d5ec1a988abf852891e1c9f0f649    558055
+gpt2_355M/model-81dee612.params     81dee612413733899f6e5fbbeac91da781805e1b    1419312986
+gpt2_774M/model-c9555788.yml        c95557880783ec4f94b09b5b045c8d9e9a198e4d    425
+gpt2_774M/model_lm-cfbfa641.params  cfbfa6419aaf1eae480fba5a1a7c8ea6096d43d6    3096157676
+gpt2_774M/gpt2-396d4d8e.merges      396d4d8ec90cb02f4d56e049e0e4add868bcd943    456318
+gpt2_774M/gpt2-9dc62091.vocab       9dc620913410d5ec1a988abf852891e1c9f0f649    558055
+gpt2_774M/model-9917e24e.params     9917e24e89c651793adea69042d6cceddfc7973c    3096150714
+gpt2_1558M/model-3f8e3175.yml       3f8e3175b9faad1cc95df2f542b486698aeb5665    425
```

Comment:
I think it should be safe to remove these `.yml` files.

Reply:
The reason is that we are directly using functions to represent the structure.

Reply:
This cfg file is used in get_pretrained_gpt2

Reply:
I mean to only remove the `gpt2_1558M/model-3f8e3175.yml`.

Reply:
The reason is that we have these configurations written in the python file: https://github.com/dmlc/gluon-nlp/blob/0223686dbc25bae416dfa764dc94ef2f6664fb00/src/gluonnlp/models/gpt2.py#L89-L97